client/MentorPanel/src/redux/actions.js

```diff
@@ -22,73 +22,57 @@ export const loadMentor = mentor => (dispatch) => {
   })
 }
 
-export const loadQuestions = (mentor_id, recommended) => (dispatch) => {
+export const loadQuestions = (mentor_id, recommended) => async (dispatch) => {
   const questions_url = questionsUrl(mentor_id)
+  const results = await papaParseAsync(questions_url)
```

Comment:
should we handle the error case here? (for async/await ops that would be try/catch)